api/v1alpha1/service_fence.proto

```diff
@@ -83,6 +83,10 @@ message ServiceFenceSpec {
     map<string, RecyclingStrategy> host = 1;
     // Switch to render servicefence as sidecar
     bool enable = 2;
+    // services in these namespaces are all stable dependency
+    repeated string namespace = 3;
+    // services match these selectors are all stable dependency
+    map<string, string> selector = 4;
```

Comment:
what about `repeat map<string, string> selectors`? Though I'm not sure whether pb supports that.

Reply:
change to labelSelector.selectors with repeated selector (map[string]string)